tests/lib/prepare-project.sh

```diff
@@ -237,6 +277,9 @@ if [ -z "$SNAPD_PUBLISHED_VERSION" ]; then
       fedora-*|opensuse-*)
          build_rpm
          ;;
+      arch-*)
+         build_pkg
```

Comment:
Can we please call this `build_arch_pkg`, people may find this confusing as `package` is a generic term.

Reply:
renamed